lava_test_plans/testcases/templates/android-xts.yaml.jinja2

```diff
@@ -26,5 +27,5 @@
         RESULTS_FORMAT: "{{xts_result_format}}"
         ANDROID_VERSION: "{{ANDROID_VERSION}}"
         TEST_REBOOT_EXPECTED: "{{xts_expect_reboot}}"
-      name: "{{test_name}}"
+      name: "{{TEST_SUITE_NAME}}"
```

Comment:
Why is it needed to be overridable here?

Reply:
it would make the use of the testcases flexible.
As you can see, if TEST_SUITE_NAME is overridable before, it would work with setting TEST_SUITE_NAME to android-vts-kernel-v7a or android-vts-kernel-v8a before this change.
That way other work could go ahead instead of being blocked by the merge of this change.
And it does not change anything for the current functions, should be no harm there.

But if you have any concerns on this, please let me know.

Reply:
The test_name should be the same as the filename, and that then mean that we don't have to do a TEST_SUITE_NAME here to be overridden.

Reply:
something like this in the files that sets the test_name  =)

set test_name = self._TemplateReference__context.name|replace('.yaml', '')

Reply:
> The test_name should be the same as the filename, 

That is true, I know that.
> and that then mean that we don't have to do a TEST_SUITE_NAME here to be overridden.

Like I said above, overriding would make it more flexible for some cases instead of waiting for the fix to be merged and deployed, and no harm for the current functions. so it's a better feature.
but obviously you don't agree with that.
Anyway, that's OK, let me drop that commit to only have the test_name fix change for this PR.